src/Microsoft.Health.Dicom.Api/Modules/SecurityModule.cs

```diff
@@ -42,9 +44,16 @@ public void Load(IServiceCollection services)
                 .AddJwtBearer(options =>
                 {
                     options.Authority = _securityConfiguration.Authentication.Authority;
-                    options.Audience = _securityConfiguration.Authentication.Audience;
                     options.RequireHttpsMetadata = true;
                     options.Challenge = $"Bearer authorization_uri=\"{_securityConfiguration.Authentication.Authority}\", resource_id=\"{_securityConfiguration.Authentication.Audience}\", realm=\"{_securityConfiguration.Authentication.Audience}\"";
+                    options.TokenValidationParameters = new TokenValidationParameters
+                    {
+                        ValidAudiences = new[]
+                        {
+                            _securityConfiguration.Authentication.Audience,
+                            GenerateSecondaryAudience(),
```

Comment:
The audience could be the appid, in which case it's just a GUID.

Reply:
What do you then think about just updating the semantics of the audience to understand a delimited list. Such that a setting of `https://example.com;https://example.com;1234` would results in the following array:

```json
[
   "https://example.com",
   "https://example.com",
   "1234"
]
```